packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -708,7 +748,8 @@ $ms-Persona-presenceSizeXl: 28px;
       background-color: $ms-color-presence-busy-average;
       transform: rotate(-45deg);
       position: absolute;
-      top: 5px;
+      // Using 40% as the top position of the cross bar element causes it to render in the correct position, even on very small persona sizes.
+      top: 40%;
```

Comment:
This seems rather arbitrary. What if the customer changes the height of the tile?